### PR TITLE
[PLAY-636] add controlsInvisible option

### PR DESCRIPTION
**Pull request by mcm69**
With this option, one can hide the controls but still click to play/pause the animation.

@shamansir plz review

### CODE REVIEW

**Comment by shamansir** [comment]
Honestly I don't think we need making controls invisible for that, I have an issue in YouTrack to handle click on canvas when controls are disabled and there's no interactivity, as a signal to start the scene. So clicking on canvas will start the movie — it's just one mouseclick handler instead of an additional canvas. It won't, probably, support pausing, though. @skavish what do you think?

**Comment by mcm69** [comment]
I don't think handling events on canvas is a good idea, all responsibility regarding that is implemented in controls and should stay there.

**Comment by shamansir** [comment]
Ok, let's see what Dima will say. 

My point also is that for API end-user the "invisible controls" option would look strange, since user doesn't know the controls canvas is placed over the whole player area, user [probably] thinks it's just a button in the center and a tiny line at the bottom, and they become invisible, which makes no sense.

Also, when controls are disabled (as there's such option), there's of course should be a way to start a scene if auto-play is also off. It's just a call to `play()`/`stop()`, `Controls` class is mostly responsible for visualizing progress and buttons which, in their turn, control playback.

File: src/anm/ui/controls.js [hunk 1]
```diff
@@ -154,6 +155,10 @@ Controls.prototype.render = function(gtime) {
         coords = state.mpos,
         time = state.time = player.state.time;
 
+    if (this.invisible) {
```
Comment (shamansir) [inline src/anm/ui/controls.js]: I think it should be above both `this.rendering = true;` and `this.checkFade(dt)`.

**Comment by skavish** [comment]
frankly I don't understand the argument. I checked the code and I don't see any extra canvas. what are we talking about here?

**Comment by mcm69** [comment]
Anton is thinking of _player_ canvas handling the clicks when controls are disabled/hidden, I think this should not be its responsibility.

**Comment by shamansir** [comment]
That's not actually _extra_ canvas, it's just not disabled and still added to DOM when it's not shown. I suppose when user _disables) controls, and auto-play is off (by default), she still expects she'll be able to run it by clicking on the player canvas. I think there's no need in special option for that and no need to load controls only for that purpose.

**Comment by mcm69** [comment]
the user does not give a damn how many elements the player creates nor it should be his business

**Comment by shamansir** [comment]
(but Yura disabled the rendering if "invisible" flag is true, it worths mentioning)

**Comment by shamansir** [comment]
This flag is a part of an API, it could be just the naming problem, as an API-user I don't actually get what "invisible controls" means.

**Comment by shamansir** [comment]
Okay, I'll merge it for the sake of release)

**Pull request merged**